Public/Src/IDE/VsCode/BuildXL.IDE.VsCode.dsc

```diff
@@ -89,23 +115,24 @@ namespace LanguageService.Server {
                             subfolder: a`projectManagement`,
                             contents: globR(d`client/projectManagement`)
                         },
+                        {
+                            subfolder: a`node_modules`,
+                            contents: [ Deployment.createDeployableOpaqueSubDirectory(VsCode.Client.installRootDir, r`node_modules`) ]
+                        },
+                        {
+                            subfolder: a`out`,
+                            contents: [ VsCode.Client.compileOutDir ]

```

Comment:
Clarification: if instead of passing `VsCode.Client.compileOutDir` we tried to pass
```typescript
Deployment.createDeployableOpaqueSubDirectory(VsCode.Client.installRootDir, r`node_modules`) 
```
we'd run into problems, because even though `out` is indeed inside of `VsCode.Client.installRootDir`, its content wasn't produced as part of the `VsCode.Client.installRootDir` opaque directory (`installRootDir` is produced by `npm install` and `out` is produced by `npm run compile`)